src/topp/FileInfo.cpp

```diff
@@ -427,34 +397,27 @@ class TOPPFileInfo : public TOPPBase
     {
       vector<FASTAFile::FASTAEntry> entries;
       FASTAFile file;
-      vector<FASTAFile::FASTAEntry>::iterator loopiter;
-      vector<FASTAFile::FASTAEntry>::iterator iter;
 
       map<char, int> aacids;
-      std::map<char, int>::iterator it;
       int number_of_aacids = 0;
 
-      mu.before();
+      SysInfo::MemUsage mu;
       //loading input
       file.load(in, entries);
-
-      mu.after();
-      std::cout << "\n\nMemory usage while loading: " << mu << std::endl;
+      std::cout << "\n\n" << mu.delta("loading FASTA") << std::endl;
 
       os << "Number of sequences: " << entries.size() << "\n"
          << "\n";
 
-      for (loopiter = entries.begin(); loopiter != entries.end(); loopiter = std::next(loopiter))
+      for (auto loopiter = entries.begin(); loopiter != entries.end(); ++loopiter)
```

Comment:
for (const auto& entriy : entries) ? (I did not see *loopiter beeing changed in the small code snipped I see...)